spec/legacy/functional/user_mailer_spec.rb

```diff
@@ -323,7 +322,7 @@
       issue = FactoryGirl.create(:work_package)
       user  = FactoryGirl.create(:user, mail: 'foo@bar.de', language: 'de')
       FactoryGirl.create(:user_preference, user: user, others: { no_self_notified: false })
-      ActionMailer::Base.deliveries.clear
```

Comment:
This one might fail, assuming that the `create` calls above issued mail delivery that is cleared here deliberately.

Ignore this comment if you ran the legacy specs locally and verified that it works as expected.